docs/REST API/rest-api.md

```diff
@@ -7015,6 +7077,53 @@ Deletes the session specified by ID. When the user associated with the session n
   "count": 43
 }
 ```
+
+### Get software title
+
+Returns the information of the specified software title. By default `versions` are sorted in descending order by the `hosts_count` field.
+
+`GET /api/v1/fleet/software/titles/{id}`
+
+#### Example
+
+`GET /api/v1/fleet/software/titles/12`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "software_title": {
+    "id": 12,
+    "title": "Firefox.app",
+    "versions_count": 3,
```

Comment:
```suggestion
```

I think we can cut this. We have the `vulnerabilities` in the `versions` array below.